src/index.ts

```diff
@@ -365,15 +359,14 @@ export class EsbuildServerlessPlugin implements ServerlessPlugin {
     }
 
     // include any "extras" from the individual function "patterns" section
-    for (const fnName in this.functions) {
-      const fn = this.serverless.service.getFunction(fnName);
+    for (const [functionAlias, fn] of Object.entries(this.functions)) {
```

Comment:
Small refactor of existing code. `fnName` was confusing because it's actually the function's alias whereas fn.name is actually the name.